book/website/collaboration/organising-conference.md

```diff
@@ -55,6 +55,7 @@ A few subsections are marked as "Relevant for all phases", as those aspects shou
 - Create possibilities for people to ask for support to attend your event.
 - Create a reporting guideline for any questions, concerns or breaches of the Code of Conduct.
 - Describe how you intend to create an inclusive event, for example, will there be a sign language interpreter or live transcription.
+- Consider allocating a budget related to the Accessibility Fund that can be dedicated to addressing transportation needs. 
```

Comment:
Is an accessibility fund mentioned anywhere else in this chapter/related chapters? I checked this markdown file and couldn't see anything at first glance, but maybe it's elsewhere? 

If that's not the case: Maybe worth rephrasing to include the idea of an accessibility fund in the first place?